src/commands/subcommands/community/handleSuggest.ts

```diff
@@ -35,9 +35,10 @@ export const handleSuggest: CommandHandler = async (
       return;
     }
 
-    const suggestionChannel = guild.channels.cache.find(
-      (el) => el.id === config.suggestion_channel
-    ) as TextChannel;
+    const suggestionChannel =
+      guild.channels.cache.find(
+        (el) => el.id === config.suggestion_channel)
+       as TextChannel | ForumChannel;
```

Comment:
```suggestion
    const suggestionChannel = guild.channels.cache.find(
      (el) => el.id === config.suggestion_channel
    ) as TextChannel | ForumChannel;
```

The formatting is breaking this apparently. We'll still need to update the actual logic for sending the suggestion.